services/src/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/kafka/RecordHandler.java

```diff
@@ -969,13 +970,11 @@ private void dumpMeters(SwitchId switchId, String correlationId, String replyToT
             producerService.sendMessageAndTrack(replyToTopic, correlationId, infoMessage);
         } catch (UnsupportedSwitchOperationException e) {
             String messageString = "Not supported: " + switchId;
-            logger.error(messageString, e);
```

Comment:
I think we should left logger

Reply:
+1

Reply:
This case is not error anymore, add info level logging